src/yachtzee.ts

```diff
@@ -673,9 +675,11 @@ export function yachtzeeChainDiet(simOnly?: boolean): boolean {
     (3 - get("_sweatOutSomeBoozeUsed", 0));
   const spleenAvailable = currentSpleenLeft + filters;
   const organsAvailable = fullnessAvailable + inebrietyAvailable + spleenAvailable;
-  const sufficientOrgans =
+  const sufficientOrgansFor30 =
     organsAvailable - synthCastsToCoverRun - extroSpleenSpace >= 30 + 5 * toInt(havePYECCharge);
-  const baseYachtzeeTurns = sufficientOrgans ? 30 : 10;
+  const sufficientOrgansFor20 =
+    organsAvailable - synthCastsToCoverRun - extroSpleenSpace >= 20 + 5 * toInt(havePYECCharge);
```

Comment:
If we plan on adding more than these 2 chain lengths, i would structure these two into an object. Something like
```
const sufficientOrgans = { 20: [long mathy expression], 30: [slightly different expression] };
```

Alternately, SLAP this logic out into another function. I think that would be nice because instead of using the nested ternary on line 682, you can use early returns.

Hell, actually,


```
const cleanableSpleen = organsAvailable - synthCastsToCoverRun - extroSpleenSpace; // Rename this variable to something better if you can 
const sufficientOrgansFor = (yachtzees: number) => cleanableSpleen  >= yachtzees + adventureExtensionBonus;
```
I really do not like the use of `toInt(boolean)` here, and great news, you don't need to use it, we already have a variable for it.